common/common.go

```diff
@@ -505,6 +582,88 @@ func containerEngineVersionCheck(containerEngineType string) error {
 	return nil
 }
 
+func isSupportedByDefaultBuildxPlatforms(containerEngineType string, platforms []string) bool {
+
+	// Only allow single platform for default builder instance
+	// multiarch platform build may fail for default builder instance
+	if len(platforms) > 1 {
+		return false
+	}
+	var args []string
+	args = append(args, "buildx")
+	args = append(args, "inspect")
+
+	out, err := exec.Command(containerEngineType, args...).Output()
```

Comment:
nit: is there an advantage to the way we have here over this?
```
	out, err := exec.Command(containerEngineType, "buildx", "inspect").Output()
```

Reply:
Changed for the ones where number of params are less than 4.